cypress/integration/BoardingPassesInfo.js

```diff
@@ -0,0 +1,67 @@
+// @noflow
+/* global
+  Cypress,
+  cy
+*/
+/* eslint-disable jest/valid-expect */
+
+describe('Boarding Passes Info', () => {
+  beforeEach(() => {
+    cy.visit('/');
+    cy.get('[data-cy=btn-existent-booking]').click();
+    cy.get('[data-cy=link-kiwi-login]').click();
+
+    cy.get('[data-cy=input-email]').type(Cypress.env('TEST_USER_EMAIL'));
+    cy.get('[data-cy=input-password]').type(Cypress.env('TEST_USER_PASSWORD'));
+    cy.get('[data-cy=btn-sign-in]').click();
+  });
+
+  it('should display boarding passes info', () => {
+    cy.get('[data-cy=btn-boarding-passes]').click();
+
+    cy
+      .get('.boardingPassesCard')
+      .find('p')
+      .contains(
+        'Here, you can download your boarding passes or see when they will become available.',
+      )
+      .get('.boardingPassesRow')
+      .should('exist');
+  });
+
+  it('should show a specific FAQ', () => {
+    cy
+      .get('[data-cy=btn-boarding-passes]')
+      .click()
+      .get('[data-cy=faq-article-link]')
+      .contains('PNR numbers');
+  });
+
+  it('should show the previous root category after clicking home', () => {
+    cy
+      .wait(2000)
```

Comment:
why wait? Cypress waits by default, so usually it's not needed

Reply:
It was because of a very strange behaviour, without wait the test was breaking locally. I was also checking the perex of the FAQ category, and there was a difference in a comma between the expected and the actual value. In the initial render and after clicking home, we are querying the GraphQL server with different section variables, and for some reason for the same category the perex differs in one character. However it's not totally clear for me, why the wait solved that. 
🤔🤯 Now I'm comparing just the titles there.